packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogWrapper.kt

```diff
@@ -21,6 +21,17 @@ import java.lang.IllegalArgumentException
  */
 @Suppress("ComplexInterface", "TooManyFunctions")
 interface DatadogWrapper {
+    /**
+     * Enables RUM for logs.
+     * By default, RUM is enabled for logs.
+     */
+    var bundleWithRum : Boolean
+
+    /**
+     * Enables Traces for logs.
+     * By default, Traces is enabled for logs.
+     */
+    var bundleWithTraces : Boolean
```

Comment:
since this is declared on the `DatadogWrapper` and not on `DdLogs`, I think it will be useful to explicitly specify here that this is for logs integration only, by using naming like `bundleLogsWithRum` / `bundleLogsWithTraces`